packit_service/service/api/projects.py

```diff
@@ -78,6 +78,41 @@ def get(self, forge, namespace, repo_name):
         return response_maker(project_info)
 
 
+@ns.route("/<forge>")
+@ns.param("forge", "Git Forge")
+class ProjectsForge(Resource):
+    @ns.expect(pagination_arguments)
+    @ns.response(HTTPStatus.PARTIAL_CONTENT.value, "Projects list follows")
+    @ns.response(HTTPStatus.OK.value, "OK")
+    def get(self, forge):
+        """List of projects of given forge"""
+
+        result = []
+        first, last = indices()
+
+        projects_list = GitProjectModel.get_forge(first, last, forge)
```

Comment:
what's the expectation for the `forge` argument? `https://github.com` or `github.com`?

I'm wondering what would be the most convenient way for API users to use the endpoint